src/Core/Rexl.Bind/Operations/BuiltinFunctions.cs

```diff
@@ -378,6 +378,10 @@ private BuiltinFunctions()
         AddOne(TextTrimFunc.TrimStart, new Sig(S.AboutTrimStart, A.Create(S.ArgSource, S.AboutTrimStart_Source)));
         AddOne(TextTrimFunc.TrimEnd, new Sig(S.AboutTrimEnd, A.Create(S.ArgSource, S.AboutTrimEnd_Source)));
 
+        AddOne(TextPadFunc.PadCenter, new Sig(S.AboutPadCenter, A.Create(S.ArgSource, S.AboutPad_Source), A.Create(S.ArgSource, S.AboutPad_Len)));
+        AddOne(TextPadFunc.PadStart, new Sig(S.AboutPadStart, A.Create(S.ArgSource, S.AboutPad_Source), A.Create(S.ArgSource, S.AboutPad_Len)));
+        AddOne(TextPadFunc.PadEnd, new Sig(S.AboutPadEnd, A.Create(S.ArgSource, S.AboutPad_Source), A.Create(S.ArgSource, S.AboutPad_Len)));
```

Comment:
The second `A.Create` first arg needs need to be the name of the length parameter, not source parameter.